helpers.js

```diff
@@ -171,8 +171,20 @@ exports.fetchOembed = async endpoint => {
   return response.data;
 };
 
-exports.selectPossibleOembedLinkNodes = markdownAST => {
-  return select(markdownAST, "paragraph link:only-child");
+exports.selectPossibleOembedLinkNodes = (markdownAST, usePrefix = false) => {
+  if (usePrefix === true) {
+    const nodes = select(markdownAST, "inlineCode");
+    var res = [];
+    nodes.map(node => {
```

Comment:
This line will get all nodes with `{type: 'inlineCode'}`. Then, I filltering it and then push to `res` array